docs/during/timetool.md

```diff
@@ -31,93 +31,7 @@ Before the time tool can be employed it must be calibrated. During the process o
-
-**Procedure**:
-
-1. Determine the run, or runs, for which you would like to generate timetool corrections. Any number of contiguous runs can be analyzed in a single task submission, as long as all the runs in the specified range were collected with the **same nominal time delay**.
-2. Create a new task, ```timetool_correct```, in the btx YAML file. This task takes four parameters:
-    - ```runs``` : A single run or range of runs to correct with the time tool. Ranges should be specified with a hyphen. E.g. 17-21
-    - ```nominal_ps``` : The nominal delay in picoseconds.
-    - ```model``` : The file location of the time tool calibration model coefficients. If left empty, btx will try to retrieve the most recent model. If none is available it will return the nominal time for each event with an error message in the logs. Refer to [this page](/before/timetool) for information on defining relative paths. Leaving this parameter blank is a fine choice, as long as you have completed a timetool calibration task.
-    - ```figs``` : True/False - Diagnostic figures will be generated. This option is case-sensitive - **capitalize** the first letter.
-
-
-    A completed entry may look like the following:
-
-```yaml
-timetool_correct:
-  runs: 15-60
-  nominal: 1
-  model:
-  figs: True
-```
-
-!!! danger "Case-sensitivity"
-    Certain parameters, such as ```figs``` above may take case-sensitive values.
-
-<ol start=3>
-<li>Submit your task using <code>elog_submit.sh</code>.</li></ol>
-```bash
-$ elog_submit.sh -c ../yamls/myyaml.yaml -t timetool_correct
-```
-<ol start=4>
-<li>Repeatedly submit the task with different sets of runs as needed. Monitor diagnostic figures to ensure that timing is not drifting.</li></ol>
-
-
-As with calibration, results will be stored in two subdirectories of the ```timetool``` folder. Figures will be stored in the ```figs``` sudirectory. The ```corrections``` subdirectory will contain a file ```r00{run}.out``` where ```{run}``` is the run number, or range of runs, analyzed with a single task submission. Results in this file are stored in a two-column, space-delimited format. Each row of the file contains a timetool correction for a single event. A uniqe event identifier of the format ```seconds-nanoseconds-fiducial``` is followed by the timetool corrected time stamp (i.e. the actual delay).
-```
-timetool/
-├── corrections/
-│   └── r00{run}.out
-└── figs/
-     └── EdgesHist_r00{run}.png
-```
+1. That the timing does not drift so far that the timetool can no longer correct the jitter. This can generally be corrected for with a simple move of the delay stage, without further modification of the timetool alignment or processing.
+2. That the results of the timetool are reasonable. As mentioned, they should be, however, diagnositics, such as the detected edge position distributions, can help determine early on if there is a need to revisit the timetool calibration, or even address an alignment issue.
```

Comment:
typo: `diagnostics`

Reply:
TODO: document how to do these two things in practice. A short note for now that we can replace with a description of the elog dashboard once we have it.